test/sslapitest.c

```diff
@@ -9599,7 +9599,10 @@ static int test_load_dhfile(void)
         goto end;
 
     SSL_CONF_CTX_set_ssl_ctx(cctx, ctx);
-    SSL_CONF_CTX_set_flags(cctx, SSL_CONF_FLAG_CERTIFICATE | SSL_CONF_FLAG_SERVER | SSL_CONF_FLAG_FILE);
+    SSL_CONF_CTX_set_flags(cctx,
+                           SSL_CONF_FLAG_CERTIFICATE
+                           | SSL_CONF_FLAG_SERVER
+                           | SSL_CONF_FLAG_FILE);
```

Comment:
I'd probably keep the SSL_CONF_FLAG_CERTIFICATE on the previous line (and update the alignment of the following lines accordingly). But yeah, this is OK as is.